src/components/BottomNavBar.tsx

```diff
@@ -1,56 +1,67 @@
-import { Flex, Text, Center, Avatar, useBoolean } from '@chakra-ui/react';
-import { Link } from 'react-router-dom';
+import { Flex } from '@chakra-ui/react';
+import { AiFillHome } from 'react-icons/ai';
+import { FaPeopleGroup } from 'react-icons/fa6';
+import { IoIosAddCircle } from 'react-icons/io';
+import { HiPaperAirplane } from 'react-icons/hi2';
 import { ROUTES } from '../constants/routes';
+import { getLocalStorage } from '../utils/storage';
+import AvatarImage from './common/AvatarImage';
+import PrimaryLink from './common/PrimaryLink';
 
-function BottomNavBar() {
-  const [loginFlag] = useBoolean(true); // 임시로 (네브바 : 로그인 버튼) 중 뭐가 보일지 결정하는 상태입니다
+const BottomNavBar = () => {
+  // context를 뭘 사용해야할지 고민
+  // user정보를 context로 하면 로그인 되어있는지 id를 받을수 있음
+  const isUser = getLocalStorage('token', '');
```

Comment:
```suggestion
  const userToken = getLocalStorage('token');
```

initialValue로 이미 빈 값인 ''을 받아오도록 되어있기 때문에 두 번째 인자는 제거해도 될 것 같습니다!!
그리고 `userToken`과 같은 이름으로 바꾸는 것이 나을 것 같습니다. `isUser`은 user인지 아닌지를 판단하는 것으로 해석이 될 수 있다고 생각해서요!!